great_expectations/data_context/util.py

```diff
@@ -183,6 +183,8 @@ def mask_db_url(cls, url: str, use_urlparse: bool = False, **kwargs) -> str:
         """  # noqa: E501
         if url.startswith("DefaultEndpointsProtocol"):
             return cls._obfuscate_azure_blobstore_connection_string(url)
+        elif cls._is_config_str(url):
```

Comment:
Can we call `str_contains_config_template` instead of defining this new function? It seems like that's what `validate_template_str_format` does and then it raises instead of returning `False`.

Reply:
Done. TBH I feel like it reads better with the helper, but I'm fine either way!